src/aiida/storage/psql_dos/orm/computers.py

```
@@ -122,7 +122,17 @@ def delete(self, pk) -> None:
         try:
             session = self.backend.get_session()
             row = session.get(self.ENTITY_CLASS.MODEL_CLASS, pk)
+            row_uuid = row.uuid
             session.delete(row)
             session.commit()
         except SQLAlchemyError as exc:
             raise exceptions.InvalidOperation(f'Unable to delete the requested computer: {exc}')
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)